drizzlepac/runsinglehap.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+""" runsinglehap.py - Module to control processing of single-visit mosaics
+
+:License: :doc:`LICENSE`
+
+USAGE: runsinglehap [-d] inputFilename
+
+Python USAGE:
+    python
+    from drizzlepac import runsinglehap
+    runsinglehap.process(inputFilename,debug=True)
+
+The '-d' option will run this task in DEBUG mode producing additional outputs.
+
```

Comment:
I would move this line up to be beneath the command line USAGE.

Reply:
Fixed (along with another typo in the doc strings) in c569ae17